common/util.go

```diff
@@ -238,6 +246,11 @@ func GetTLSConfig(config *apicommon.TLSConfig) (*tls.Config, error) {
 		}
 	}
 
+	if !config.InsecureSkipVerify && len(caCertPath)+len(clientCertPath)+len(clientKeyPath) == 0 {
```

Comment:
This is redundant. When the code is executed here, `InsecureSkipVerify` must be false, and line 250 already has the check there.

Reply:
yes, just wanted to create a separate test for it but since the default value is false, doesn't make much sense. Need to remove the test as well. 
Done!